includes/wikia/models/DesignSystemCommunityHeaderModel.class.php

```diff
@@ -475,4 +501,61 @@ private function formatLocalNavData( array $items, int $nestingLevel ): array {
 	private function getSpecialPageURL( $name ): string {
 		return SpecialPage::getTitleFor( $name )->getLocalURL();
 	}
+
+	private function isFandomStoreCommunity( $wikiId ) {
+		global $wgFandomStoreMap;
+		return array_key_exists( $wikiId, $wgFandomStoreMap );
+	}
+
+	private function getFandomStoreData() {
```

Comment:
Let's add a "fromCache" to the method name for clarity.

Reply:
Updated naming to 
- getFandomStoreDataFromCache
- getFandomShopDataFromIntentX